roles/freeswitch/tasks/main.yml

```diff
@@ -127,6 +28,20 @@
         comment: FreeSWITCH open source softswitch
         groups: freeswitch
 
+    - name: Download freeswitch binaries
+      get_url:
+        url: "{{ freeswitch_binaries_download_url }}"
+        dest: /tmp/freeswitch.tar.gz
```

Comment:
I think the tar should be stored outside of /tmp/ otherwise freeswitch would get reinstalled after a server restart

Reply:
Ah, that's right, thanks! I will change that.